mri/operators/fourier/cartesian.py

```diff
@@ -88,18 +95,26 @@ def op(self, img):
             images the coils dimension is put first
         """
         if self.n_coils == 1:
-            return self._mask * np.fft.ifftshift(np.fft.fftn(
+            return self.mask * np.fft.ifftshift(np.fft.fftn(
                                     np.fft.fftshift(img), norm="ortho"))
         else:
             if self.n_coils > 1 and self.n_coils != img.shape[0]:
                 raise ValueError("The number of coils parameter is not equal"
                                  "to the actual number of coils, the data must"
                                  "be reshaped as [n_coils, Nx, Ny, Nz]")
             else:
-                # TODO: Use joblib for parallelization
-                return np.asarray([self._mask * np.fft.ifftshift(np.fft.fftn(
-                                    np.fft.fftshift(img[ch]), norm="ortho"))
-                                   for ch in range(self.n_coils)])
+                axes = tuple(np.arange(1, img.ndim))
```

Comment:
I find this weird.
If for example `img` has 2 dimension, this will be `np.arange(1, 2)`, giving `array([1])`, which is not what we want, either for the `fftn` or for the `fftshift`.

Or am I missing something?

Reply:
ndim = dimension of image + 1 for multichannel case.

Reply:
ofc makes sense